_data/simulations/coral_6a/meta.yaml

```diff
@@ -0,0 +1,54 @@
+_id: 0dc3d1f0-76fd-11e7-a3e9-cb63094aa20f
+metadata:
+  author: Andrea Jokisaari
+  email: andrea.jokisaari@northwestern.edu
+  github_id: amjokisaari
+  simulation_name: coral_6a
+  summary: Upload-a-thon August 2017 test upload
+  timestamp: '31 July, 2017'
+  implementation:
+    repo:
+      url: 'https://www.dropbox.com/s/qusutz8g24hx54q/free_energy.csv'
+      version: '00000'
+    container_url: ''
+  hardware:
+    architecture: x86_64
+    parallel_model: distributed
+    clock_rate: '2.7'
+    cores: '6'
+    nodes: '1'
+  software:
+    name: moose
+    version: 'MOOSE hash 24ad223, Coral hash b49ae39'
+    repo:
+      url: 'https://github.com/idaholab/moose'
+      version: '00000'
+benchmark:
+  version: '0'
+  id: 6a
+data:
+  - name: free_energy
+    url: 'https://www.dropbox.com/s/qusutz8g24hx54q/free_energy.csv'
```

Comment:
The above comments really apply to this URL, not the metadata. My test builds fail to show a free energy plot because this URL is not accessible from my machine.